context/app/static/js/components/savedLists/SavedListPanel/SavedListPanel.jsx

```diff
@@ -2,7 +2,13 @@ import React from 'react';
 import { Panel } from 'js/shared-styles/panels';
 
 function getEntityCounts(listSavedEntities) {
-  const counts = { Donor: 0, Sample: 0, Dataset: 0 };
+  const counts = new Proxy(
+    // If a "Support" entity appears, we want to count it, but it shouldn't be included otherwise.
+    { Donor: 0, Sample: 0, Dataset: 0 },
+    {
+      get: (target, name) => (name in target ? target[name] : 0),
+    },
+  );
```

Comment:
```suggestion
  const counts = {};
```

Reply:
Right now, we always get counts for Donor, Sample, and Dataset, even if they are zero. I think with this suggestion, we'd only see non-zero counts... Does that seem ok to you?

Reply:
Whoops, the initial counts should be `counts = {Donor: 0, Sample: 0, Dataset: 0};` then.